modules/gateway/peers.go

```diff
@@ -604,3 +604,20 @@ func (g *Gateway) Peers() []modules.Peer {
 	}
 	return peers
 }
+
+// Online returns true if the node is connected to the internet. During testing
+// we always assume that the node is online
+func (g *Gateway) Online() bool {
+	if build.Release != "standard" {
```

Comment:
Should instead check whether it equals dev or testing. We're going to be adding a testnet release constant soon, and I don't want to miss this patch of code. It's much less noticeable if you are accidentally always saying online if you can actually be both online or offline vs. always saying offline when you shouldn't be.

Reply:
basically we're inverting this because if we mess up after adding future constants, it'll be more noticeable that we messed up.